crates/cli/src/exec.rs

```diff
@@ -103,10 +107,13 @@ pub fn exec_dry_run<Arg, Builder: HostEnvBuilder<Arg = Arg>>(
     wasm_binary: Vec<u8>,
     phantom_functions: Vec<String>,
     arg: Arg,
+    config: Builder::HostConfig,
 ) -> Result<()> {
     let loader =
         ZkWasmLoader::<Bn256, Arg, Builder>::new(zkwasm_k, wasm_binary, phantom_functions)?;
-    loader.run(arg, true, false)?;
+    let result = loader.run(arg, config, true, false)?;
+    println!("total guest instructions used {:?}", result.guest_statics);
```

Comment:
use `info!()` if possible

Reply:
I have moved some println from circuit_with_witness out side to cli in exec.rs